gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -98,6 +112,17 @@ public synchronized int read() throws IOException {
     return (b & 0xff);
   }
 
+  private void captureDuration(String suffix, Stopwatch stopwatch, int byteRead) {
```

Comment:
why not capture the duration taken by close() as well?

Reply:
It was not of interest but can add it for `close` as well.

Reply:
This can be interesting in the write() case.

Reply:
Added.